### PR TITLE
🏗 input agnostic pipeline

### PR DESCRIPTION
**Pull request by dmiller15**
resolves: https://github.com/kids-first/kf-genomics-harmonization/issues/465

### CODE REVIEW

**Comment by dmiller15** [comment]
OK added the blub to the README.

File: readme.md [hunk 1]
```diff
@@ -48,4 +48,49 @@ this can be used later on for further analysis in joint trio genotyping and subs
 
 Note, for all vcf files, indexing may be required - a "secondary file" requirement.
 
-![WF Visualized](./kfdrc_alignment_wf.png?raw=true "Workflow diagram")
\ No newline at end of file
+![WF Visualized](./kfdrc_alignment_wf.png?raw=true "Workflow diagram")
```
Comment (migbro) [inline readme.md:51]: Please update this with how it looks now.  You can save the image from rabix - might be easiest to convert from svg to png and crop.

File: readme.md [hunk 2]
```diff
@@ -48,4 +48,49 @@ this can be used later on for further analysis in joint trio genotyping and subs
...
+
+### Tips for running:
+1. For the fastq input file lists (PE or SE), make sure the lists are properly ordered. The items in
+   the arrays are processed based on their position. These lists are dotproduct scattered. This means
+   that the first file in `input_pe_reads_list` is run with the first file in `input_pe_mates_list`
+   and the first string in `input_pe_rgs_list`. This also means these arrays must be the same
+   length or the workflow will fail.
+1. Must have these associated indexes:
+    - knownsite vcfs: Each file requires a `'.tbi'` index
+    - reference fasta: BWA, picard, and samtools indexes (`'.64.amb', '.64.ann', '.64.bwt',
```
Comment (migbro) [inline readme.md:80]: Both sets of bwa indices are n't actually needed, just one or the other

**Comment by migbro** [comment]
Please fix the ancient `samtools_covert_to_cram` typo in this workflow, the tool, and references to it in other workflows. It's not your fault, but now is a good opportunity to do so!

**Comment by migbro** [comment]
In workflow reorg, just move the `deprecated` up one, perhaps can make it called `deprecated_workflows`, but just make it at the same level as current tools and workflows as having an extra level will break tool sym links

**Comment by dmiller15** [comment]
Yeah those dev files are old and unused. Just scraps of code if people want to read as examples.

**Pull request merged**